packages/fluent/scss/menu/_variables.scss

```diff
@@ -0,0 +1,322 @@
+/// The background gradient of hover menu items.
+/// @group menu
+$kendo-menu-item-hover-gradient: $kendo-menu-item-gradient !default;
+/// The shadow of hover menu items.
+/// @group menu
+$kendo-menu-item-hover-shadow: $kendo-menu-item-shadow !default;
+
+// Menu items expanded
+
+/// The text color of expanded menu items.
+/// @group menu
+$kendo-menu-item-expanded-text: $kendo-menu-item-text !default;
+/// The background color of expanded menu items.
+/// @group menu
+$kendo-menu-item-expanded-bg: get-theme-color-var( neutral-30 ) !default;
+/// The border color of expanded menu items.
+/// @group menu
+$kendo-menu-item-expanded-border: $kendo-menu-item-border !default;
+/// The background gradient of expanded menu items.
+/// @group menu
+$kendo-menu-item-expanded-gradient: $kendo-menu-item-gradient !default;
+/// The shadow of expanded menu items.
+/// @group menu
+$kendo-menu-item-expanded-shadow: $kendo-menu-item-shadow !default;
+
+// Menu items focused
+
+/// The text color of focused menu items.
+/// @group menu
+$kendo-menu-item-focused-text: $kendo-menu-item-text !default;
+/// The background color of focused menu items.
+/// @group menu
+$kendo-menu-item-focused-bg: $kendo-menu-item-bg !default;
+/// The border color of focused menu items.
+/// @group menu
+$kendo-menu-item-focused-border: $kendo-menu-item-border !default;
+/// The background gradient of focused menu items.
+/// @group menu
+$kendo-menu-item-focused-gradient: $kendo-menu-item-gradient !default;
+/// The outline of focused menu items.
+/// @group menu
+$kendo-menu-item-focused-outline: get-theme-color-var( neutral-130 ) !default;
+/// The shadow of focused menu items.
+/// @group menu
+$kendo-menu-item-focused-shadow: $kendo-menu-item-shadow !default;
+
+
+// Menu popup
+
+/// Horizontal padding of the menu popup.
+/// @group menu
+$kendo-menu-popup-padding-x: $kendo-popup-padding-x !default;
+
+/// Vertical padding of the menu popup.
+/// @group menu
+$kendo-menu-popup-padding-y: $kendo-popup-padding-x !default;
+
+/// Width of the border around the menu popup.
+/// @group menu
+$kendo-menu-popup-border-width: $kendo-popup-border-width !default;
+
+/// Font sizes of the menu popup.
+/// @group menu
+$kendo-menu-popup-font-size: var( --kendo-font-size-md, inherit ) !default;
+$kendo-menu-popup-font-size-sm: $kendo-menu-popup-font-size !default;
+$kendo-menu-popup-font-size-md: $kendo-menu-popup-font-size !default;
+$kendo-menu-popup-font-size-lg: $kendo-menu-popup-font-size !default;
+
+/// Line heights used along with $kendo-font-size.
+/// @group menu
+$kendo-menu-popup-line-height: var( --kendo-line-height-md, 30px ) !default;
+$kendo-menu-popup-line-height-sm: $kendo-menu-popup-line-height !default;
+$kendo-menu-popup-line-height-md: $kendo-menu-popup-line-height !default;
+$kendo-menu-popup-line-height-lg: $kendo-menu-popup-line-height !default;
+
+/// The background of the menu popup.
+/// @group menu
+$kendo-menu-popup-bg: $white !default;
```

Comment:
it's better to use kendo-component-bg color here. Check if kendo-component-color and border can be used as well instead of initial. In this way once the dark swatch is created, the menu will look okay.